plugins/embed/gui-themes/themes/zeit-amp/item/source/soundcloud.dust

```diff
@@ -0,0 +1,13 @@
+{>"themeBase/item/source/base"/}
+{<itemClass}soundcloud{/itemClass}
+{<contentCore}
+	{+contentExternalContainer}{/contentExternalContainer}
+	{+contentAttr}{/contentAttr}
+{/contentCore}
+{<contentExternal}
+	<amp-soundcloud height="166" layout="fixed-height" data-trackid="{Meta.id}"></amp-soundcloud>
```

Comment:
In the head of the document (created at www.zeit.de). We are just embedding the liveblog HTML inside our documents using [ESI](https://en.wikipedia.org/wiki/Edge_Side_Includes). All CSS and corresponding scripts will get referenced in the template head from our backend.